.github/workflows/scripts/formatScan/clangtidy.sh

```diff
@@ -11,7 +11,7 @@ log_path=${log_dir}/clangtidy.log
 cd ${REPO_DIR}
 mkdir build
 cd build
-cmake .. -G Ninja -DNS_USE_CLANG_TIDY=CHECK -DBTLA_USE_OPENMP=OFF
+cmake .. -G Ninja -DNS_USE_CLANG_TIDY=CHECK -DBTLA_ENABLE_OPENMP=OFF -DNS_USE_OMP=OFF
```

Comment:
`-DNS_USE_OMP=OFF` only should be enough

Reply:
the scan uses the existing code repo. the build folder may have CMakeCache file already. If it has BTLA_ENABLE_OPENMP=ON, the clang-tidy will use it too.

Reply:
No. The `build` directory is just created 2 lines before this line. (Without the `-p` flag, `mkdir` will exit with error when trying to create something already exists.)